dev/ColorPicker/ColorPicker.cpp

```diff
@@ -701,17 +710,17 @@ void ColorPicker::OnThirdDimensionSliderValueChanged(winrt::IInspectable const&
     {
     case winrt::ColorSpectrumComponents::HueValue:
     case winrt::ColorSpectrumComponents::ValueHue:
-        s = m_thirdDimensionSlider.Value() / 100.0;
+        s = m_thirdDimensionSlider.get().Value() / 100.0;
```

Comment:
Should there be a if (auto&&  thirdDimensionSlider = m_thirdDimensionSlider.get()) check first?

Reply:
Not needed, since this is an event handler for thirdDimensionSlider.

---
In reply to: [401110313](https://github.com/microsoft/microsoft-ui-xaml/pull/2191#discussion_r401110313) [](ancestors = 401110313)